tests/unit/Codeception/Module/FacebookTest.php

```diff
@@ -117,21 +116,9 @@ public function testLoginToFacebook()
         // preconditions: #3 test user logged in on facebook
         $this->module->haveTestUserLoggedInOnFacebook();
 
-        // go to our page with facebook login button
-        $browserModule->amOnPage('/facebook');
```

Comment:
why you've deleted those tests statements ?
 now it's not testing that facebook login work